extensions/Gemini/python/README.md

```diff
@@ -0,0 +1,8 @@
+# Using Gemini with AIConfig
+
+We are prioritizing the Python library Usage
+
+1. Install `pip install aiconfig-extensions-gemini`
+2. Import the extension `import aiconfig_extension_gemini`
+
+And you're all good to go! This extension automatically updates the model parser registry with the model "Gemini-pro". Check out the Gemini Cookbook in the aiconfig repo for a full example.
```

Comment:
We should make sure we're consistent with the model name between this and workbooks, for when we download as aiconfig, right? cc @saqadri 

Also, should this be `gemini-pro` (lowercase)?

Reply:
This could also be updated in the adapter logic if its not consistent. There are already a couple of differences in aiconfig vs workbooks right now.

Reply:
It should be lowercased here too